helix-core/src/main/java/org/apache/helix/controller/GenericHelixController.java

```diff
@@ -172,11 +173,15 @@
   private boolean _inMaintenanceMode;
 
   /**
-   * The timer that can periodically run the rebalancing pipeline. The timer will start if there is
-   * one resource group has the config to use the timer.
+   * The executors that can periodically run the rebalancing pipeline. A
+   * SingleThreadScheduledExecutor will start if there is one resource group has the config to do
+   * periodically rebalance.
```

Comment:
We totally only have one SingleThreadScheduledExecutor at most for each genericHelixController object, right? This sounds like per resource group has one. Can we make it clear the life cycle of SingleThreadScheduledExecutor is the same as genericHelixController?

Reply:
TFTR. Updated.